WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -255,7 +255,13 @@ extension PostCoordinator: Uploader {
                 return
             }
 
-            posts.forEach() { self.retrySave(of: $0 ) }
+            posts.forEach() { post in
+                let shouldRetry = post.status == .draft && (!post.hasRemote() || post.original?.status == .draft)
```

Comment:
@diegoreymendez This works great. 

@maxme gave a good suggestion that we can make this safer. Android only has this condition:

```swift 
let shouldRetry = post.status == .draft && !post.hasRemote()
```

The `post.original?.status == .draft` can cause a published post to be reverted back to draft. Here's an example scenario where I was able to reproduce this:

1. iOS: Create a `DRAFT` post and upload it.
2. iOS: Go offline.
3. Web: Publish the post. 
4. iOS: While still offline, edit the post and save it. Do not publish. 
5. iOS: Go online and let the auto-uploader finish. 

When the upload is done, the post is reverted back to a draft.

Reply:
For context: as we discussed privately this is going to always be an issue unless we get API support.

But if we add a timeout to these auto-uploads, they're going to be much safer to allow.  So we'll scale back this solution to what @shiki proposed until we can get more safeguards in place.

The decision is:

1. Upload local drafts only in the PR.
2. Discuss the timer and other possible safeguards as soon as possible.